website/docs/r/packet_capture.html.markdown

```diff
@@ -0,0 +1,188 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_packet_capture"
+sidebar_current: "docs-azurerm-resource-network-packet-capture"
+description: |-
+  Configures Packet Capturing against a Virtual Machine using a Network Watcher.
+
+---
+
+# azurerm_packet_capture
+
+Configures Packet Capturing against a Virtual Machine using a Network Watcher.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name = "packet-capture-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_network_watcher" "test" {
+  name                = "network-watcher"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name = "production-network"
```

Comment:
Minor nitpick: should `terraform fmt` the examples to make them easier to read

Reply:
done